src/service/report-service.ts

```diff
@@ -307,15 +313,15 @@ export default abstract class ReportService extends WithManager {
      * @param forId - The user ID to get the totals for
      * @param fromDate - The from date to get the totals for (inclusive)
      * @param tillDate - The till date to get the totals for (exclusive)
-     * @returns {Promise<{ totalExclVat: Dinero.Dinero, totalInclVat: Dinero.Dinero }>} - The totals
+     * @returns {Promise<{ totalExclVat: Dinero.Dinero, totalInclVat: Dinero.Dinero, transactionCount: number }>} - The totals
      */
-  public async getTotals(forId: number, fromDate: Date, tillDate: Date): Promise<{ totalExclVat: Dinero.Dinero, totalInclVat: Dinero.Dinero }> {
+  public async getTotals(forId: number, fromDate: Date, tillDate: Date): Promise<{ totalExclVat: Dinero.Dinero, totalInclVat: Dinero.Dinero, transactionCount: number }> {
     const query = this.manager.createQueryBuilder(ProductRevision, 'productRevision')
       .innerJoin('productRevision.vat', 'vatGroup')
       .innerJoin(SubTransactionRow, 'subTransactionRow', 'subTransactionRow.productProductId = productRevision.productId AND subTransactionRow.productRevision = productRevision.revision')
       .innerJoin(SubTransaction, 'subTransaction', 'subTransaction.id = subTransactionRow.subTransactionId')
-      .innerJoin(Transaction, 'transaction', 'transaction.id = subTransaction.transactionId');
-    this.addSelectTotals(query);
+      .innerJoin(Transaction, 'transaction', 'transaction.id = subTransaction.transactionId')
+      .addSelect('COUNT(DISTINCT transaction.id)', 'transaction_count');
```

Comment:
This is not the same. I meant adding the count to the addselecttoals function, you seem to have removed it?